src/android/notification/Builder.java

```diff
@@ -116,17 +120,33 @@ public Notification build() {
             return new Notification(context, options);
         }
 
-        Uri sound     = options.getSound();
+        // Forked for Komed Health custom sounds
+        // For pre-oreo device we need to use setSound method in NotificationBuilder
+        // Uses a default Komed sound by default, or priority sound if "priority" option given.
+        Uri soundUri = Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" + context.getPackageName() + "/raw/res\\raw\\komed_notification_default");
+        String channelId = Manager.DEFAULT_CHANNEL_ID;
+
+        String sound = options.getSound();
+        if (sound != null)
+            if (sound.equalsIgnoreCase("priority")) { // for priority notification sound
+                soundUri = Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" + context.getPackageName() + "/raw/komed_notification_priority");
+                if (SDK_INT >= O)
+                    channelId = Manager.PRIORITY_CHANNEL_ID;
+            } else if (!sound.isEmpty()) { // for other notification sounds
```

Comment:
just to check, so what if option for example "1" or "true" or "foobar" is given - would it still use the komed default sound or fall into this set custom uri block?

Reply:
It will fall into the custom uri block for all cases except `null`
"1" or "true" or "foobar" will go to custom uri as well.